### PR TITLE
[admin_v3] Update all table styles

### PR DESCRIPTION
**Pull request by dacook**
#### What? Why?

This updates the styles for all tables in the admin interface (behind feature toggle), according to the new design. 

- Further applies new styling for #10630

One example: 
Screen Shot 2023-06-08 at 3 23 59 pm
Screen Shot 2023-06-08 at 3 24 12 pm

More context: while preparing for the new admin product table, I applied some of the new styles across all tables (because we want to do this anyway). I've extracted this from [the closed PR](https://github.com/openfoodfoundation/openfoodnetwork/pull/10916).

#### What should we test?
The style changes are pretty low-risk changes, behind a feature toggle, and will continue to be refined before release, so I think we don't need to test for functionality at this point.

#### What's changed?
This includes some changes to background/foreground colours, which we might need to resolve before releasing the feature toggle.
* odd/even table rows used to have different background colours, now they are mostly all white
* pagination area used to be white, now it is grey (to accomodate white pagination buttons).
* pagination buttons are now white with barely any border, but they should always appear inside the grey pagination area.

Other changes include:
* base font colour has gone from the old blue to near-black (`#191C1D`). This changes the font colour inside tables, but also form labels and maybe other things.
* Remove uppercasing of form elements and titles. They now appear mostly in sentence case, although I noticed an odd one out.

#### Release notes

Changelog Category: Feature toggle 😎 

The title of the pull request will be included in the release notes.

#### Documentation updates

* After merging, we can update the link here to point to `master`: https://github.com/openfoodfoundation/openfoodnetwork/wiki/Design-system

### CODE REVIEW

**Comment by dacook** [comment]
Screen Shot 2023-06-07 at 3 16 13 pm
Screen Shot 2023-06-07 at 3 16 22 pm
Screen Shot 2023-06-07 at 3 16 38 pm
Screen Shot 2023-06-07 at 3 16 48 pm
Screen Shot 2023-06-07 at 3 17 36 pm
Screen Shot 2023-06-07 at 3 17 59 pm
Screen Shot 2023-06-07 at 3 18 21 pm

**Comment by mariocarabotta** [comment]
_odd/even table rows used to have different background colours, now they are all white_ tweaking the spacing between row should fix readability on this
_pagination buttons are now white with barely any border_ I think this is aligned with the [new style of pagination](https://github.com/openfoodfoundation/openfoodnetwork/issues/10877), should be ok
_pagination area used to be white, now it is grey (to accomodate white pagination buttons)_ see above
_base font colour has gone from the old blue to near-black (#191C1D). This changes the font colour inside tables, but also form labels and maybe other things._ this is a good thing. there's really no reason to use blue as a base colour for text
_Remove uppercasing of form elements and titles. They now appear mostly in sentence case, although I noticed an odd one out._ sentence case is more readable than uppercase (except in very rare occasions), I'll keep an eye on that to see if I spot other ones

**Comment by dacook** [comment]
I’ve increased borders to 4px and fixed up the header alignment. Latest screenshot: 
Screen Shot 2023-06-07 at 5 03 16 pm

**Comment by dacook** [comment]
A few more tweaks and let's go!
* update colour for all headers (change dark blues to "Sherpa blue)
* also update column header colours
* increase left border to match
* fix bottom border

Screen Shot 2023-06-08 at 3 23 59 pm
Screen Shot 2023-06-08 at 3 24 12 pm

File: .prettierignore [hunk 1]
```diff
@@ -9,6 +9,7 @@
 babel.config.js
 postcss.config.js
 
+/app/webpacker/css/admin_v3/globals/
```
Comment (jibees) [inline .prettierignore:12]: I'm wondering why you've added the file here? You're happy with the existing indentation and prettier will erase that?

File: app/webpacker/css/admin_v3/globals/palette.scss [hunk 2]
```diff
@@ -5,3 +5,5 @@ $color-3: #008397 !default; // Teal (Allports)
 $color-4: #6788a2 !default; // Dark Blue
 $color-5: #c85136 !default; // Red/Orange (Mojo)
 $color-6: #ff9300 !default; // Yellow
+$color-7: #EFF1F2 !default; // Light grey
```
Comment (jibees) [inline app/webpacker/css/admin_v3/globals/palette.scss:8]: This file is no more prettified now ;( (I can see because of the uppercase on html color code)

**Review by jibees — APPROVED** [review]
Looks good to me. Two concerns:

 - I think we should prettier ignore only file, not folder. 
 - I'm afraid about resolving conflicts with our PRs ... ;)

File: .prettierignore (hunk 1 above)
Comment (jibees) [reply .prettierignore:12]: Can we ignore that particular file? Instead of ignoring the whole folder?

Comment (dacook) [reply .prettierignore]: Good point, thanks I've updated this!

**Review by jibees — APPROVED** [review]

**Review by rioug — APPROVED** [review]
Looks good :+1:

**Comment by rioug** [comment]
Do we need @mariocarabotta review on this as well ?

**Comment by dacook** [comment]
Thanks, Mario will review further once it has been released and can be reviewed with real-life data.

I've fixed conflicts and will merge once specs pass.

**Pull request merged**